### PR TITLE
Add com.alientrap.nexuiz-classic

### PR DESCRIPTION
**Pull request by rszibele**
It would be nice to have the original Nexuiz available on Flathub.

### CODE REVIEW

**Comment by rszibele** [comment]
bot, build com.alientrap.nexuiz-classic

**Comment by rszibele** [comment]
hmm, don't have permission to stop the first build.
bot, build com.alientrap.nexuiz-classic

File: com.alientrap.nexuiz-classic.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            <image type="source" width="1366" height="768">https://raw.githubusercontent.com/rszibele/flathub/com.alientrap.nexuiz-classic/screenshots/screenshot_3.jpg</image>
+        </screenshot>
+    </screenshots>
+    <url type="homepage">http://www.alientrap.com/games/nexuiz/</url>
+    <url type="bugtracker">https://sourceforge.net/projects/nexuiz/</url>
+    <provides>
+        <binary>nexuiz</binary>
+    </provides>
+    <developer_name>Alientrap</developer_name>
+    <categories>
```
Comment (bilelmoussaoui) [inline com.alientrap.nexuiz-classic.metainfo.xml:55]: Should be in the desktop file only. They are merged together later

File: com.alientrap.nexuiz-classic.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        <category>Game</category>
+        <category>ArcadeGame</category>
+        <category>Shooter</category>
+    </categories>
+    <keywords>
+        <keyword>FPS</keyword>
+        <keyword>Quake</keyword>
+    </keywords>
```
Comment (bilelmoussaoui) [inline com.alientrap.nexuiz-classic.metainfo.xml:63]: Same as for categories

File: com.alientrap.nexuiz-classic.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+app-id: com.alientrap.nexuiz-classic
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
```
Comment (bilelmoussaoui) [inline com.alientrap.nexuiz-classic.yaml:3]: Any reason for not using 19.08?

**Comment by rszibele** [comment]
@bilelmoussaoui Thanks for the quick review! I've made the changes. My personal Flatpaks still use 18.08 so I've mistakenly used it instead of 19.08. I've tested it and it works without issues, so I've changed it to 19.08.

File: com.alientrap.nexuiz-classic.yaml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  build-commands:
+  - mkdir -p /app/bin/
+  - mkdir -p /app/share/nexuiz/
+  - mv * /app/share/nexuiz/
+  sources:
+  - type: archive
+    url: https://sourceforge.net/projects/nexuiz/files/NexuizRelease/Nexuiz 2.5.2/nexuiz-252.zip
+    sha512: da661eb2ff1764fed1d5f5547b89e4f9aa47613d41a3999c0202d8858ec2b882e4ef0ec182a81d1bdb25dff2ef65aad6af132009884e3557206941a736c2a68c
+
+- name: launcher-i386
```
Comment (bilelmoussaoui) [inline com.alientrap.nexuiz-classic.yaml:49]: i386 isn't supported anymore starting from 19.08

File: com.alientrap.nexuiz-classic.desktop [hunk 5]
```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Name=Nexuiz Classic
```
Comment (bilelmoussaoui) [inline com.alientrap.nexuiz-classic.desktop:2]: Should be upstreamed if possible. The same as the screenshots/metainfo file

**Comment by bilelmoussaoui** [comment]
Have you contacted the upstream developers if they were interested in (co-)maintaing their app on Flathub?

File: com.alientrap.nexuiz-classic.yaml (hunk 4 above)
Comment (rszibele) [reply com.alientrap.nexuiz-classic.yaml:49]: Ah, that's unfortunate, but inevitable.

**Comment by rszibele** [comment]
I doubt they would accept any of the changes upstream or even (co-)maintain the game, as the game hasn't seen a new release since 2009, but I will contact them. I'll remove the i386 support in a bit.

**Comment by bilelmoussaoui** [comment]
bot, build com.alientrap.nexuiz-classic

File: com.alientrap.nexuiz-classic.desktop [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+Comment[de]=Ein netzwerkfähiges 3D Ego-Shooter Spiel mit aufwändigen Grafikeffekten
+Comment[fr_FR]=Un Ego-Shooter en 3D pouvant être joué en réseau
+Comment[pl_PL]=Gra sieciowa 3D, strzelanka, w pierwszej osobie z efektami graficznymi wysokiej jakosci
+Comment[ro_RO]=Un joc 3D cu împuşcături, perspectivă personală şi efecte grafice de înaltă calitate
+Exec=/app/bin/launcher.sh
+Icon=com.alientrap.nexuiz-classic
+Terminal=false
+Type=Application
+Categories=Game;ArcadeGame;Shooter
+Keywords=FPS,Quake
```
Comment (TingPing) [inline com.alientrap.nexuiz-classic.desktop:13]:
```suggestion
Keywords=FPS;Quake;
```

File: com.alientrap.nexuiz-classic.desktop [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+Comment=A networked 3D first person shooter game with high-quality graphic effects
+Comment[de]=Ein netzwerkfähiges 3D Ego-Shooter Spiel mit aufwändigen Grafikeffekten
+Comment[fr_FR]=Un Ego-Shooter en 3D pouvant être joué en réseau
+Comment[pl_PL]=Gra sieciowa 3D, strzelanka, w pierwszej osobie z efektami graficznymi wysokiej jakosci
+Comment[ro_RO]=Un joc 3D cu împuşcături, perspectivă personală şi efecte grafice de înaltă calitate
+Exec=/app/bin/launcher.sh
+Icon=com.alientrap.nexuiz-classic
+Terminal=false
+Type=Application
+Categories=Game;ArcadeGame;Shooter
```
Comment (TingPing) [inline com.alientrap.nexuiz-classic.desktop:12]:
```suggestion
Categories=Game;ArcadeGame;Shooter;
```

File: com.alientrap.nexuiz-classic.yaml [hunk 8]
```diff
@@ -0,0 +1,75 @@
+app-id: com.alientrap.nexuiz-classic
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: launcher.sh
+finish-args:
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
```
Comment (barthalion) [inline com.alientrap.nexuiz-classic.yaml:9]: Does it actually work on Wayland?

Comment (rszibele) [reply com.alientrap.nexuiz-classic.yaml]: No, only through XWayland. I've removed it.

**Comment by rszibele** [comment]
bot, build com.alientrap.nexuiz-classic

**Pull request closed without merging**